doc/usermacros.rst

```diff
@@ -6,7 +6,7 @@ User macro reference
    :param DIRECTORY: Directory containing messages.  May be absolute or
      relative to ``CMAKE_CURRENT_SOURCE_DIR``.
    :param FILES:  Files containing messages, relative to `msgdir`
-   :param PACKAGE:  Optional alternate packagename (if the current project doesn't match the
+   :param BASE_DIR:  Optional alternate packagename (if the current project doesn't match the
```

Comment:
Ok . so I should remove it ? The PACKAGE option was useful no ? It allowed
messages in a foo project to actually create headers as if they were part
of a foo_msgs package (or anything else).
Le 3 sept. 2015 00:25, "Dirk Thomas" <notifications@github.com> a écrit :

> In doc/usermacros.rst
> <https://github.com/ros/genmsg/pull/59#discussion_r38592847>:
>
> > @@ -6,7 +6,7 @@ User macro reference
> >     :param DIRECTORY: Directory containing messages.  May be absolute or
> >       relative to ``CMAKE_CURRENT_SOURCE_DIR``.
> >     :param FILES:  Files containing messages, relative to `msgdir`
> > -   :param PACKAGE:  Optional alternate packagename (if the current project doesn't match the
> > +   :param BASE_DIR:  Optional alternate packagename (if the current project doesn't match the
>
> The user should probably never use it. It is only used by actions
> internally:
> https://github.com/ros/genmsg/blob/indigo-devel/cmake/genmsg-extras.cmake.em#L90-L93
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ros/genmsg/pull/59/files#r38592847>.
>